lib/features/modeling/util/ModelingUtil.js

```diff
@@ -1,18 +1,24 @@
+import { isString } from 'min-dash';
+
 export { is, isAny } from '../../../util/ModelUtil';
 
 import { isAny } from '../../../util/ModelUtil';
 
+/**
+ * @typedef {import('../../../model/Types').BpmnElement} BpmnElement
```

Comment:
One thought: Shall we rename `BpmnElement` to `Element`? It is clear that all `Element` dealt with in `bpmn-js` are `BPMN` types, derived from diagram elements.

Reply:
👍🏻